src/main/java/bio/terra/cli/serialization/userfacing/UFResourceCloneDetails.java

```diff
@@ -0,0 +1,138 @@
+package bio.terra.cli.serialization.userfacing;
+
+import bio.terra.cli.businessobject.Resource;
+import bio.terra.cli.utils.UserIO;
+import bio.terra.workspace.model.CloneResourceResult;
+import bio.terra.workspace.model.CloningInstructionsEnum;
+import bio.terra.workspace.model.ResourceCloneDetails;
+import bio.terra.workspace.model.StewardshipType;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import java.io.PrintStream;
+import java.util.Optional;
+import java.util.UUID;
+import javax.annotation.Nullable;
+import org.apache.commons.lang.StringEscapeUtils;
+
+@JsonDeserialize(builder = UFResourceCloneDetails.Builder.class)
+public class UFResourceCloneDetails {
```

Comment:
I wondered if this shouldn't be a nested class in `UFClonedWorkspace`. Might add too much complexity to e.g. building to be worth it, but it would provide some encapsulation.